presets/svelte-webpack/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import type {
+  WebpackConfiguration,
+  StorybookConfig as StorybookConfigBase,
+  TypescriptOptions as BaseTypescriptOptions,
+} from '@storybook/core-webpack';
+
+export type { BuilderResult } from '@storybook/core-webpack';
+
+export type TypescriptOptions = BaseTypescriptOptions;
+
+export type StorybookConfig<TWebpackConfiguration = WebpackConfiguration> =
+  StorybookConfigBase<TWebpackConfiguration>;
+
+export type SvelteOptions = {
```

Comment:
How does this get associated with any of the config types? Or is that not necessary?

Reply:
It gets composed here:
https://github.com/storybookjs/storybook/blob/63460cf5a18d72dbff4cec20c01f4082c3135b5f/frameworks/svelte-webpack5/src/types.ts#L15-L17